frontend/src/flashcards/Flashcard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import FlashcardFront from './FlashcardFront';
+import FlashcardBack from './FlashcardBack';
+import ReactCardFlip from 'react-card-flip';
+
+/*
+  This is the styling for the main flashcard.
+*/
+const flipStyle ={
```

Comment:
We might have to reconfigure our pull requests later because I do remember seeing a lot of this code before in a different pull request.